lib/perf_hooks.js

```diff
@@ -59,6 +59,14 @@ class Performance extends EventTarget {
       timeOrigin: this.timeOrigin,
     }, opts)}`;
   }
+
+  toJSON() {
+    return {
+      nodeTiming: this.nodeTiming.toJSON(),
+      timeOrigin: this.timeOrigin,
+      eventLoopUtilization: this.eventLoopUtilization()
+    };
```

Comment:
should we use this instead? because Performance is a static class

```js
  toJson: {
    configurable: true,
    enumerable: false,
    value: toJson,
  },
```

Reply:
I am not sure since other performance classes are also following the same practice and I implemented it in the same fashion.

Reply:
It is enumerable in Chrome:

![image](https://user-images.githubusercontent.com/2352663/111861171-8b266080-894c-11eb-992d-90688698be2e.png)